Libplanet/Net/Swarm.cs

```diff
@@ -435,13 +435,24 @@ public void Dispose()
             }
 
             IEnumerable<BoundPeer> peers = seedPeers.OfType<BoundPeer>();
+            IEnumerable<BoundPeer> peersBeforeBootstrap = RoutingTable.Peers;
 
             await PeerDiscovery.BootstrapAsync(
                 peers,
                 pingSeedTimeout,
                 findNeighborsTimeout,
                 depth,
                 cancellationToken);
+
+            if (!((NetMQTransport)Transport).Running)
```

Comment:
It seems to force using `NetMQTransport` as `Transport`. How about checking with `as` keyword and `?` nullable operator or introducing `Running` into `ITransport`?

First one I said as code...

```suggestion
            if (!(Transport as NetMQTransport)?.Running)
```

Second one I said as code...

```csharp
// Swarm.cs
            if (!Transport.Running)
```
```csharp
// ITransport.cs
     public bool Running { get; }
```

Reply:
I'll chose to add `Running` into `ITransport`.